codegen/test_data/bar.json

```diff
@@ -157,7 +157,7 @@
 					],
 					"IsEndpoint": false,
 					"QueryParamGoStatements": null,
-					"ReqHeaderFields": {},
+					"ReqHeaderGoStatements": null,
```

Comment:
Let's be careful when updating edge-gateway, this will break a lot of existing configs